internal/services/machinelearning/machine_learning_workspace_resource.go

```diff
@@ -94,6 +95,39 @@ func resourceMachineLearningWorkspace() *pluginsdk.Resource {
 
 			"identity": commonschema.SystemAssignedUserAssignedIdentityRequired(),
 
+			"kind": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"Default",
+					"FeatureStore",
+				}, false),
+				Default: "Default",
+			},
```

Comment:
should this be exposed or is it internal arm details? or is it expected the user can configure kind?

Reply:
This can be exposed. Initially MLW had only one type. Later the service added a new type `FeatureStore` and set the previous type of MLW to `Default`.

Reply:
trying to trigger the label remove action